.travis.yml

```diff
@@ -127,7 +127,7 @@ jobs:
     - git config --global user.name "Travis CI"
     - git remote set-url --push origin "git@github.com:$TRAVIS_REPO_SLUG"
     - export ${!TRAVIS*}
-    - sphinx-versioning push docs docs . -- -b dirhtml  -A html_theme=rasabaster
+    - sphinx-versioning push docs docs . -- -b dirhtml  -W --keep-going -A html_theme=rasabaster
```

Comment:
mhm not sure if this is a good idea, because I think this will run the warning check for all versions that get build, but we really only want to check the current one - we should have a separate stage for this as before

Reply:
yes, i see what you mean, it failed because it tested other branches. Do you mean a separate stage like [this](https://github.com/RasaHQ/rasa/pull/3435#pullrequestreview-238566196)? Alan was saying he didn't think he had put the `-W` in the right place 😄

It may be easiest if you write the stage

Reply:
yes I'll add it :+1: